### PR TITLE
chore: upgrade styleguidist

### PR DESCRIPTION
**Pull request by drinchev**

### CODE REVIEW

**Comment by frabbit** [comment]
is this customization stuff part of the new styleguidist or was that already possible before?

**Comment by drinchev** [comment]
Was already possible before I guess, but I didn't want to do it before I upgrade the styleguidist as well.

File: styleguide/ComponentsList.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          href,
+          content,
+          shouldOpenInNewTab,
+        }) => {
+          const isItemSelected = windowHash === href;
+          return (
+            <li key={href}>
+              <a
+                className={classnames(
+                  'olt-Link',
```
Comment (frabbit) [inline styleguide/ComponentsList.js]: let's use variables here: oltStyles.Link

File: styleguide/Markdown/Pre.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PreRenderer = ({ className, children }) => {
+  const isHighlighted = className && className.indexOf('lang-') !== -1;
+  return (
+    <div className="olt-Card pre--wrapper">
```
Comment (frabbit) [inline styleguide/Markdown/Pre.js]: let's use variables here: oltStyles.Card

File: styleguide/Markdown/Pre.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      <div className="olt-Card-content">
```
Comment (frabbit) [inline styleguide/Markdown/Pre.js]: let's use variables here: oltStyles.CardContent

File: styleguide/Markdown/Pre.js (hunk 2 above)
Comment (frabbit) [reply styleguide/Markdown/Pre.js]: or the Card component directly

**Comment by frabbit** [comment]
It looks great in general, just a few tiny notes.

File: styleguide/Markdown/Pre.js (hunk 2 above)
Comment (drinchev) [reply styleguide/Markdown/Pre.js]: Good catch. I replaced it with `Card` component.

File: styleguide/ComponentsList.js (hunk 1 above)
Comment (drinchev) [reply styleguide/ComponentsList.js]: ✅

**Pull request merged**